pokeapp/src/components/PokemonCard.js

```diff
@@ -81,48 +85,57 @@ const PokemonCard = ({
         hoverMe();
       }}
     >
-
-      <div className="card-header">
-        <div className="name">
-          <h3>{makeUpperCase(pokemon.name)}</h3>
+      <Link to="/" style={{ textDecoration: "none" }}>
+        <div className="card-header">
+          <div className="name">
+            <h3>{makeUpperCase(pokemon.name)}</h3>
+          </div>
         </div>
-      </div>
 
-      {/* Card image starts */}
-      <div style={cardImage}>
-      {/* put buttons in here */}
-        <div className="card-btns-container">
-          <div className={applyStyles(`poke-value`)}><img src={PokeDollarIcon} alt="poke dollar image" />{pokemon.value}</div>
-          <div className={applyStyles(`poke-buy`)}
+        {/* Card image starts */}
+        <div style={cardImage}>
+          {/* put buttons in here */}
+          <div className="card-btns-container">
+            <div className={applyStyles(`poke-value`)}>
+              <img src={PokeDollarIcon} alt="poke dollar image" />
+              {pokemon.value}
+            </div>
+            <div
+              className={applyStyles(`poke-buy`)}
               onClick={(e) => {
                 e.stopPropagation();
                 dispatch(buyPokemon(pokemon));
               }}
-          >Buy</div>
-           {pokemon.quantity > 0 ? (
-            <div className={applyStyles(`poke-sell`)}
-              onClick={(e) => {
-                e.stopPropagation();
-                dispatch(sellPokemon(pokemon));
-              }}
-          >Sell</div>) : null}
+            >
+              Buy
+            </div>
+            {pokemon.quantity > 0 ? (
+              <div
+                className={applyStyles(`poke-sell`)}
+                onClick={(e) => {
+                  e.stopPropagation();
+                  dispatch(sellPokemon(pokemon));
+                }}
+              >
+                Sell
+              </div>
+            ) : null}
+          </div>
         </div>
 
-      </div>
-
-      <div className="type-id-container">
-        <div className="pokemon-quantity">
-          {pokemon.quantity > 0 ? <p>x{pokemon.quantity}</p> : ""}
+        <div className="type-id-container">
+          <div className="pokemon-quantity">
+            {pokemon.quantity > 0 ? <p>x{pokemon.quantity}</p> : ""}
```

Comment:
You could also have the last ternary conditional return `null`. Unless, you'd encountered bugs with that?